compiler/runtime.go

```diff
@@ -389,7 +389,7 @@ func (e *unaryEvaluator) Eval(ctx context.Context, scope Scope) (values.Value, e
 	if err != nil {
 		return nil, err
 	}
-	values.CheckKind(ret.Type().Nature(), e.t.Nature())
+	// values.CheckKind(ret.Type().Nature(), e.t.Nature())
```

Comment:
i don't think you meant to commit commented out code.

Reply:
We're mostly removing these checks because they are more often wrong than correct. We may re-add them so I've been commenting them out whenever they cause a problem.